src/main/resources/config.sk

```diff
@@ -289,6 +289,12 @@ databases:
 		# Variables are saved constantly no matter what is set here, thus a server crash will never make you loose any variables.
 		# Set this to 0 to disable this feature.
 
+		backups to keep: -1
+		# if enabled, will check the specified number of backup files in the backup directory each time a backup file is created
+		# If the number of backup files >= the number of 'backups to keep', it will delete files until number of files is equal to `backups to keep`
+		# If disabled, no backup files will be deleted
+		# By default, -1 equals disabled, set this to any number above 0 to enabled. Can not be set to 0
```

Comment:
Setting to 0 could disable backups too?

Reply:
@sovdeeth would you like for me to add that functionality? Where if they put 0, it doesn’t start the backup task?

Reply:
yes but i would like opinions

Reply:
I honestly feel like 0 should also disable it. Skript users sometimes don't know how configs work and would expect "0" to work and not realize -1 also works.